addons/mail/static/src/components/call_view/call_view.xml

```diff
@@ -9,20 +9,26 @@
 
             <!-- Call members display -->
             <div class="o_CallView_participantContainer d-flex justify-content-between w-100 h-100 overflow-hidden" t-on-click="callView.onClick" t-on-mousemove="callView.onMouseMove">
-                <t t-if="callView.layout !== 'tiled'">
-                    <div class="o_CallView_mainParticipantContainer d-flex flex-grow-1 justify-content-center mw-100 mh-100">
-                        <t t-if="callView.mainParticipantCard">
-                            <CallParticipantCard className="'o_CallView_participantCard'" record="callView.mainParticipantCard"/>
+                <t t-if="callView.mainParticipantCard">
+                    <div class="o_CallView_mainParticipantContainer d-flex flex-grow-1 justify-content-center position-relative mw-100 mh-100">
+                        <CallParticipantCard className="'o_CallView_participantCard'" record="callView.mainParticipantCard"/>
+                        <t t-if="callView.showOverlay">
+                            <t t-if="!callView.hasSidebar">
+                                <i class="o_CallView_sidebarButton position-absolute fa fa-arrow-left" title="Show sidebar" t-on-click="callView.onClickShowSidebar"/>
+                            </t>
+                            <t t-elif="callView.hasSidebar">
+                                <i class="o_CallView_sidebarButton position-absolute fa fa-arrow-right" title="hide sidebar" t-on-click="callView.onClickHideSidebar"/>
+                            </t>
```

Comment:
Could be shorter with `t-if` and `t-else` on the elements themselves (4 less LOCs)